package.json

```diff
@@ -8,7 +8,9 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "node test/helpers/pretest.js",
-    "test": "grunt",
```

Comment:
Assuming integration tests run faster, then I think the order should be different - run the fast tests first to get quick feedback, run the slower acceptance suite only after we verified that the building blocks work correctly.